packages/teams-components/src/components/SplitButton/SplitButton.test.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+import { render } from '@testing-library/react';
+import { SplitButton } from './SplitButton';
+import { CalendarRegular } from '@fluentui/react-icons';
+
+describe('SplitButton', () => {
+  it('should render', () => {
+    render(<SplitButton>Test</SplitButton>);
+  });
+
+  it('should throw error if menuTitle is provided without main title', () => {
+    expect(() =>
+      render(<SplitButton menuTitle="Menu">Test</SplitButton>)
```

Comment:
I don't understand this requirement, why would menu title make the title required?

Reply:
If title is provided without menu title, tooltip will be applied to the full SplitButton
If both are given, there will be separate tooltip for action button and menu button - so I think it makes no sense to be able to provide only menu title